pkg/docker/docker.css

```diff
@@ -151,10 +151,6 @@
     text-overflow: ellipsis;
 }
 
-.panel .panel-heading > span {
```

Comment:
I have removed this on purpose and added class `pull-right` to the span in index.html. The section is just adding one more definition to the docker.css that is defined in in base.css as class `pull-right`.